<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QTextEdit" name="logs">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>Port :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="portComboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="refreshButton">
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>DÃ©marrer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>274</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelMessage">
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="messageLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer vers APRS.fi :</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="aprsCheckBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Oui</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourceLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>F4LTZ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicatif source :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicatif destination : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="destLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>F4KMN-8</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
